examples/chaiscript/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+cmake_minimum_required(VERSION 3.2)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-ChaiScript)
+
+# DOCUMENTATION_START {
+hunter_add_package(chaiscript)
+find_package(chaiscript CONFIG REQUIRED)
+# DOCUMENTATION_END }
+
+if (CMAKE_HOST_UNIX) 
+    #set unix link libraries
+    list(APPEND LIBS "dl") 
```

Comment:
Instead of `dl` you should use `CMAKE_DL_LIBS`:
* https://cmake.org/cmake/help/latest/variable/CMAKE_DL_LIBS.html

Reply:
Will correct in chaiscript CMakeLists.txt.